code/modules/mob/living/carbon/monkey/life.dm

```diff
@@ -12,11 +12,18 @@
 	if(..())
 
 		if(!client)
-			if(stat == CONSCIOUS)
-				if(!handle_combat())
-					if(prob(33) && canmove && isturf(loc) && !pulledby)
+			if(stat == CONSCIOUS)	
+				if(on_fire || buckled || restrained())
+					if(!resisting && prob(MONKEY_RESIST_PROB))
+						resisting = TRUE
+						walk_to(src,0)
+						resist()

```

Comment:
This will actually fix monkeys walking in space

Reply:
so it's not just an optimization but an _improvement_? scandalous